kustomization/components/samba-upload/README.md

```diff
@@ -0,0 +1,3 @@
+# Samba Component
```

Comment:
I think this should stay in your repository since it won't be shared by me.

Reply:
Did you confirm that? I thought it'd be better for tests to upstream all the components rather than just the intersection we use. This way it's harder for changes to break my setup (especially since we don't have a good versioning story).

Reply:
> Did you confirm that?

Our mutual friend who has the same printer as me used NFS, but it looks like it can do CIFS as well, so in the interest of harmonizing our configs, I'll do the same.  I'll need to look more closely at this setup now.